packages/forms-web-app/src/controllers/appeal-householder-decision/cannot-appeal.js

```diff
@@ -16,12 +16,7 @@ const getCannotAppeal = (req, res) => {
 	);
 	let deadlinePeriod = getDeadlinePeriod(appeal.appealType, appeal.eligibility.applicationDecision);
 
-	if (arraysEqual(Object.values(deadlinePeriod), ['days', 181])) {
-		deadlinePeriod = {
-			time: 6,
-			duration: 'months'
-		};
-	} else if (arraysEqual(Object.values(deadlinePeriod), ['days', 83])) {
+	if (arraysEqual(Object.values(deadlinePeriod), ['days', 84])) {
```

Comment:
can you bring in time and duration from config here?

Reply:
There's a few places with this, might look into adding descriptive strings to the config instead